src/dev/src/app/timeline/timeline.demo.module.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+import { ClarityModule } from '@clr/angular';
```

Comment:
nit: this isn't a component that uses Clarity Angular, don't need this technically.

Reply:
Removed.

Reply:
Switched to ClrIconModule as I'm using `clr-icon` in the component.